bigbluebutton-html5/imports/ui/components/menu/component.jsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { defineMessages, injectIntl } from "react-intl";
+
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import { Divider } from "@material-ui/core";
+
+import Icon from "/imports/ui/components/icon/component";
+import Button from "/imports/ui/components/button/component";
+
+import { styles } from "./styles";
+
+const intlMessages = defineMessages({
+  close: {
+    id: 'app.dropdown.close',
+    description: 'Close button label',
+  },
+});
+
+//Used to switch to mobile view

```

Comment:
We have a [helper](https://github.com/bigbluebutton/bigbluebutton/blob/develop/bigbluebutton-html5/imports/utils/deviceInfo.js#L8) that could be used here to check for mobile devices instead of checking the window size.